src/test/java/testPackage/properties/WebTests.java

```diff
@@ -33,6 +34,7 @@ public void beforeClass() {
         baseURL = SHAFT.Properties.web.baseURL();
         browserWindowWidth = 1920;
         browserWindowHeight = 1080;
+        incognitoMode = SHAFT.Properties.web.incognitoMode();
```

Comment:
this test will break the rest of the pipeline, any changes in configuration/properties must be done before method, and must be reversed after method.